README.md

```diff
@@ -8,17 +8,12 @@ navigation: 3
 
 ## Overview
 The æternity æpp components are meant to provide æpp developers with reusable [Vue.js](https://vuejs.org) components 
-that adhere to our [styleguide](https://github.com/aeternity/aepp-prototypes). The javascript comes unbundled so
-please use a build process such as [webpack](https://webpack.js.org/), [browserify](http://browserify.org/), 
-[grunt](https://gruntjs.com/), [gulp](https://gulpjs.com/), etc.
+that adhere to our [styleguide](https://github.com/aeternity/aepp-prototypes).
 
 The github repository is [here](https://github.com/aeternity/aepp-components).
 
-## Hackathon Notice
-We've rewritten and refactored the components for better usage in the hackathon (24.11.2017). 
-Beware that the currently existing apps (like identity, aexistence/notary, wall, et cetera) 
-still use older versions of this repository. In the near future we will update these repositories 
-to use the latest aepp-components.
+__Note__: this repository is currently in active development and things might break / change in the future
+but we will be adhering to the Semantic Versioning.
```

Comment:
Can we be compatible with semantic versioning in current development?

Reply:
Thats why this PR exists: https://github.com/aeternity/aepp-components/pull/210 after this we'll start publishing in the main `@aeternity/aepp-components` package with semantic versioning as you're doing in the `@aeternity/aepp-components-3` package.

Reply:
I don't see the issue with adhering with sematic versioning in the (very near?) future.